_includes/scenarios/integrate/succeeding-attributes-to-update-their-values.md

```diff
@@ -0,0 +1,84 @@
+Its `shareInfo` stores the same `requestReference` and the peer's Address, as well as the `id` of the RepositoryAttribute, whose `content` was copied, in the field `sourceAttribute`.
+
+Concluding, sharing an IdentityAttribute will create an own shared IdentityAttribute copy for every peer you shared the Attribute with at your side and a peer shared IdentityAttribute copy for each peer at their side.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/c1195765-c1ca-46b4-9468-1735f4a018cc" id="_s0-VcxrpcCM"></iframe></div>
+
+### Succeeding a RepositoryAttribute
+
+Next, let's consider the case that the `value` of your [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) changes and you want to update it.
+To [succeed the RepositoryAttribute]({% link _docs_use-cases/use-case-consumption-succeed-a-repositoryattribute.md %}), a new [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) with the updated `content` will be created.
+This successor stores the `id` of the old version, the predecessor, in its `succeeds` property.
+The predecessor is updated, too, such that its `succeededBy` property links to the successor.
+Consequently, the different versions of a LocalAttribute created by Attribute successions make up a doubly linked list.
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/8e81cd6b-240f-4e41-ae08-9eea13b32974" id="zt0-EawwJWIx"></iframe></div>
+
+### Notifying a peer about a RepositoryAttribute succession
+
+After succeeding a RepositoryAttribute, you can check out [with which of your peers you have previously shared the succeeded Attribute]({% link _docs_use-cases/use-case-consumption-get-shared-versions-of-a-repositoryattribute.md %}) to choose those, you'd like to [notify about the succession]({% link _docs_use-cases/use-case-consumption-notify-peer-about-repositoryattribute-succession.md %}).
+Your own shared IdentityAttributes associated with the peers chosen will be succeeded in the same manner as your RepositoryAttribute before.
+Then, a [Notification]({% link _docs_integrate/data-model-overview.md %}#notification) is sent to the peers, containing a [PeerSharedAttributeSucceededNotificationItem]({% link _docs_integrate/data-model-overview.md %}#peersharedattributesucceedednotificationitem).
+In its `successorContent` property the updated IdentityAttribute is transmitted and automatically a likewise succession at their side will be triggered, such that their LocalAttribute versions replicate the succession chain at your side.
```

Comment:
Hier ist unklar was mit _it_ gemeint ist. das Attribut oder die Notification. Besser "In the notification's `successorContent` property ...".